core/src/test/scala/org/apache/spark/sql/store/TokenizationTest.scala

```diff
@@ -39,18 +39,22 @@ class TokenizationTest
   val table = "my_table"
   val table2 = "my_table2"
   val all_typetable = "my_table3"
+  var planCaching : Boolean = false
 
   override def beforeAll(): Unit = {
     // System.setProperty("org.codehaus.janino.source_debugging.enable", "true")
     System.setProperty("spark.sql.codegen.comments", "true")
     System.setProperty("spark.testing", "true")
+    planCaching = Property.PlanCaching.get(snc.sessionState.conf)
+    Property.PlanCaching.set(snc.sessionState.conf, true)
```

Comment:
Same, changing to true for all tests?

Reply:
Several tests from this suite is having assertions similar to this which starts failing when planCaching disabled:
```
val cacheMap = SnappySession.getPlanCache.asMap()
assert(cacheMap.size() == 1
```